driver/main_args.ml

```diff
@@ -77,6 +77,11 @@ let mk_dllpath f =
   "<dir>  Add <dir> to the run-time search path for shared libraries"
 ;;
 
+let mk_errorsize i =
+  "-error-size", Arg.Int i,
+  "<size>  Increase the size of error message to <size> (in arbitrary units)"
+;;
+
```

Comment:
Since its in "arbitrary units", would it make sense to give some indication of what is a reasonable value for this? And/or indicate what the default value is?

Reply:
This is a very good point. I added the default value in the `-help` description and a typical range in the various manuals.